i18n/src/parser.js

```diff
@@ -33,7 +46,12 @@ function parsePattern(pattern) {
       positive = patternParts[0],
       negative = patternParts[1];
 
-  var positiveParts = positive.split(DECIMAL_SEP),
+  // The parsing logic below assumes that there will always be a DECIMAL_SEP in the pattern.
```

Comment:
Nobody reads comments. When you want to understand the code, you look at the tests 😛 
<sub>(I'll update the comment.)</sub>